duckduckgo/Android
# Accessibility settings

Task/Issue URL: https://app.asana.com/0/488551667048375/1201006464305306/f

### Description
This PR adds new accessibility capabilities to our browser. Users can change the font size (and override system settings), and force zoom on websites.

### Steps to test this PR

_Override system font size_
1. Fresh install 
2. skip initial screens until browser
3. Visit 2 websites in different tabs
4. Go to Settings -> accessibility
5. Override system font size and set font size to 150
6. Go back to the browser
7. Ensure both sites font size changed
8. Go to Settings -> accessibility
9. Disable toggle to use system font size (don't change the font size)
10. Go back to the browser
11. Ensure both size fonf size changed again

_Force zoom_
1. Open youtube on a new tab
2. Ensure you can't pinch to zoom
2. Go to Settings -> accessiblity
3. Go back to the youtube tab
4. Ensure you can pinch to zoom (notice the tab will refresh automatically)
